sections/predictive-search.liquid

```diff
@@ -0,0 +1,58 @@
+{%- if predictive_search.performed -%}
+  <div id="predictive-search-results">
+    {%- if predictive_search.resources.products.size > 0 -%} 
+      <h3 id="predictive-search-products" class="predictive-search__heading">
```

Comment:
Swap for `h2` since this will be the direct heading below the "Search results" `h1`.